src/software/pipeline/main_incrementalSfM.cpp

```diff
@@ -137,7 +138,9 @@ int main(int argc, char **argv)
       "If the process is done on renderfarm, it doesn't make sense to wait for user inputs")
     ("useLocalBA,l", po::value<bool>(&useLocalBundleAdjustment)->default_value(useLocalBundleAdjustment),
       "Enable/Disable the Local bundle adjustment strategy.\n"
-      "It reduces the reconstruction time, especially for big datasets (500+ images).\n");
+      "It reduces the reconstruction time, especially for big datasets (500+ images).\n")
+    ("localBAGraphDistance", po::value<std::size_t>(&localBundelAdjustementGraphDistanceLimit)->default_value(localBundelAdjustementGraphDistanceLimit),
+      "Graph-distance limit setting the Active region in the Local Bundle Adjustment strategy (by default: 1).\n");
```

Comment:
Proposition:
"Graph-distance limit to define the Active Region in the Local Bundle Adjustment strategy."